.github/workflows/release-script-tests.yml

```diff
@@ -1,6 +1,7 @@
 name: Release Script Tests
 
 on:
+  workflow_call:
```

Comment:
Can you add a comment on top of this line with

```
# workflow_call is used to indicate that a workflow can be called by another workflow. 
```

Reply:
Sure thing!